service/templates/cloudformation/security_groups.yaml

```diff
@@ -0,0 +1,49 @@
+{{define "security_groups" }}
+  MasterSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription:  {{ .MasterSecurityGroupName }}
+      VpcId: {{ .VPCID }}
+      Tags:
+        - Key: Name
+          Value:  {{ .MasterSecurityGroupName }}
```

Comment:
Forgot to mention that instead of setting the security group name I'm adding a Name tag. As there are restrictions on updates if you set the name.

> If you don't specify a GroupName, AWS CloudFormation generates a unique physical ID and uses that ID for the group name. For more information, see Name Type.


> If you specify a name, you cannot perform updates that require replacement of this resource. You can perform updates that require no or some interruption. If you must replace the resource, specify a new name.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group.html